pom.xml

```diff
@@ -822,5 +822,27 @@ THE SOFTWARE.
         <failIfNoTests>false</failIfNoTests>
       </properties>
     </profile>
+    <profile>
+      <id>jdk-17-and-above</id>
+      <activation>
+        <jdk>[17,)</jdk>
+      </activation>
+      <properties>
+        <!--
+          Animal Sniffer does not work on Java 17, but we cannot remove it from our configuration
+          entirely until we drop support for Java 8. As a temporary solution, we skip it when
```

Comment:
(Well, we _could_ remove it, we choose not to.)

Reply:
I think we ought to keep it as long as we still compile releases using Java 8 (which has no `<release>` support). Hence the comment that we can drop Animal Sniffer when we drop Java 8.